Oversettelse development/data og development/ux til engelsk (#594)
Altinn/altinn-studio-docs: content/app/development/logic/instantiation/_index.en.md

}
```

### Example 2 - Instantiation only allowed for application owner

> The code base this example is based on is available [here](https://altinn.studio/repos/ttd/example-app-1).

Reviewer: ```suggestion
The application this example is based on is available [here](https://altinn.studio/repos/ttd/example-app-1).
```

}
```

> Logic connected to date handling has been added to ensure that norwegian time applies and 

Reviewer: ```suggestion
Logic connected to date handling has been added to ensure the Norwegian time zone
```

        };
```

> In addition you have the oppurtunity to use property

Reviewer: ```suggestion
In addition you have the opportunity to use property
```

Reviewer: I've found your weak spot! :imp:
